database/src/main/resources/db/migration/V1.50.0__NOTIF-603_add_org_id_email_aggregation.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE email_aggregation
+  ADD COLUMN org_id text;
```

Comment:
Could you merge the migrations?
Also I guess we would need the org_id everywhere there is an account_id

Reply:
regarding the second question. I guess you mean something like "we need to add it everywhere where we do have an accountId". Yes, that's true.

Reply:
I mean merge them together in a single file :)

Reply:
aaah. Now I got it. Yes sure :)